sensors_script/device_availability_monitor.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
Could you add detailed comments to your code where necessary but avoid parrot comments.
[https://bytedev.medium.com/code-comment-anti-patterns-and-why-the-comment-you-just-wrote-is-probably-not-needed-919a92cf6758#:~:text=Parrot%20comments&text=This%20means%20they%20don't,it%20is%20a%20good%20practice.](Check this link)